source/common/http/conn_manager_impl.cc

```diff
@@ -611,6 +611,19 @@ ConnectionManagerImpl::ActiveStream::ActiveStream(ConnectionManagerImpl& connect
 
 ConnectionManagerImpl::ActiveStream::~ActiveStream() {
   stream_info_.onRequestComplete();
+  Upstream::HostDescriptionConstSharedPtr upstream_host =
+      connection_manager_.read_callbacks_->upstreamHost();
+
+  if (upstream_host != nullptr) {
+    Upstream::ClusterRequestResponseSizeStatsOptRef req_resp_stats =
+        upstream_host->cluster().requestResponseSizeStats();
+    if (req_resp_stats.has_value()) {
+      // record request body size
```

Comment:
nit: proper grammar in comments (capitalized first letter, period)

here and below

alternatively just get rid of these comments, the code speaks for itself IMO

Reply:
Yeah, will update it.